### PR TITLE
[#156911499] Implement basic paas-admin acceptance tests

### PR DESCRIPTION
**Pull request by keymon**
https://www.pivotaltracker.com/story/show/156911499

What
----

We wan to add some integration tests for paas-admin which modify and make assertions against a real Cloud Foundry environment.

The current tests which use HTTP mocks don't accurately represent Cloud Controller's functionality. For example, they aren't able to catch the bug that prevented org managers from inviting users in #156761419. You need to do manual testing in a browser to be confident that there are no existing bugs or new regressions.

In this PR we implement the initial scaffolding for the integration test with some basic tests. We will add more tests in follow up stories 

See commits for details.

Dependencies
------------------

Once this merged, update and review https://github.com/alphagov/paas-cf/pull/1463

How to review
-------------

Code review. 

Test/Review together with https://github.com/alphagov/paas-cf/pull/1463

Who can review
---------------

Anyone

### CODE REVIEW

File: acceptance-tests/main.test.ts [hunk 1]
```diff
@@ -0,0 +1,130 @@
+import {describe,it,before,after} from 'mocha';
```
Comment (paroxp) [inline acceptance-tests/main.test.ts]: Was there any particular reason to install `mocha`? Is something missing from jest that is already installed and used?

File: acceptance-tests/main.test.ts [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+import expect from 'expect';
```
Comment (paroxp) [inline acceptance-tests/main.test.ts]: Again, additional dependency that is already introduced by jest.

File: acceptance-tests/main.test.ts (hunk 1 above)
Comment (keymon) [reply acceptance-tests/main.test.ts]: Jest did not work OK with zombie. We commented it in the commit.

File: acceptance-tests/main.test.ts (hunk 2 above)
Comment (keymon) [reply acceptance-tests/main.test.ts]: We wanted to use the jest matchers, despite using mocha.

**Comment by keymon** [comment]
> I may be on the wrong here, but you seem to have introduced new functionality that isn't really used in the application but the integration tests itself. Such creating and deleting orgs and users. I'm not sure if it's the right thing to do. Wouldn't you rather set these up and tear down outside of the application?

On one hand yes, you are right here. But on the other hand these methods are very likely useful within paas-admin or even if extract these clients.

**Comment by richardTowers** [comment]
Sorry @paroxp - I didn't see your comments when I hit merge 🤦‍♂️

**Pull request merged**